corehq/apps/cloudcare/api.py

```diff
@@ -183,8 +184,13 @@ def keys():
                 for bool in status_to_closed_flags(self.status):
                     yield [self.domain, owner_id, bool]
 
-        view_results = CommCareCase.view('hqcase/by_owner', keys=keys,
-                                         include_docs=False, reduce=False)
+        view_results = CommCareCase.view(
+            'hqcase/by_owner',
+            keys=keys,
+            include_docs=False,
+            reduce=False,
+            stale=settings.COUCH_STALE_QUERY
+        )
```

Comment:
noted, good idea.

would be really interesting if we could use `quickcache` for this. like everytime we save a case we use `redis` to store the case_id to the case for ~30 mins. then we can:
1. check to see if any of the ownerids have cases in `redis`
2. use the `hq_cases` view with stale=ok and hope that 30 mins is enough time for the view to update.